src/constructors.jl

```diff
@@ -156,9 +156,24 @@ function process_args(
   end
   inline, check_empty, u₁, u₂, v, threads, warncheckarg
 end
+# check if the body of loop is a block, if not convert it to a block issue#395
+function check_loopbody!(q)
+  if q isa Expr && q.head == :for
+    if q.args[2].head != :block
```

Comment:
```suggestion
    if q.args[2].head !== :block
```
Testing identity is likely a little faster, because it is a non-overloadable builtin, meaning there's no dispatch and it's guaranteed to return `Bool`, while type unstable `==` and `!=` can return anything. 
This isn't a big deal here, as `if` only accepts `Bool`, meaning we effectively have a type assert on the return anyway. 
So something to keep in mind when working on `Expr` more generally, as their `.args` are `Vector{Any}`. 

Otherwise, a more succinct check:
```
    if Meta.isexpr(q.args[2], :block)
```

Reply:
> So something to keep in mind when working on Expr more generally, as their .args are Vector{Any}.

Thanks for you suggestion.